src/arrayMethodSort.js

```diff
@@ -1,11 +1,40 @@
+/* eslint-disable no-unused-expressions */
 'use strict';
 
 /**
  * Implement method Sort
  */
+const compareAsStrings = (a, b) => {
+  const stringA = String(a);
+  const stringB = String(b);
+
+  if (stringA > stringB) {
+    return 1;
+  } else if (stringA === stringB) {
+    return 0;
+  } else {
+    return -1;
+  }
```

Comment:
```suggestion
  } 
    return -1;
```
This 'else' is redundant. Any way this code will run only if two previous conditions are false